tests/installation/multipath.pm

```diff
@@ -19,7 +19,19 @@ use testapi;
 sub run {
     # Sometimes multipath detection takes longer
     assert_screen "enable-multipath", 60;
-    send_key "alt-y";
+    # This module is supposed to be invoked only if MULTIPATH is true
+    # or MULTIPATH_DIALOG_YES is set
+    if (get_var("MULTIPATH") or get_var("MULTIPATH_DIALOG_YES")) {
```

Comment:
`multipath.pm` is invoked when is same condition matched in `main_common.pm`. Isn't  it duplication?

Reply:
No, it is not the same. (cf. my comment below for main_common.pm). Case distinction here is intended based not on whether MULTIPATH_DIALOG_YES is defined or not, but rather on its value:

- MULTIPATH_DIALOG_YES == true (e.g. 1) ==> expect "activate multipath?" dialogue and press "Yes"
- MULTIPATH_DIALOG_YES == false (e.g. 0) ==> expect "activate multipath?" dialogue and press "No"
- (MULTIPATH_DIALOG_YES is undef ==> do not expect this dialogue; main_common.pm not to invoke multipath.pm in the first place)